thoth/adviser/boots/solved_software_environment.py

```diff
@@ -78,7 +81,18 @@ def run(self) -> None:
         _LOGGER.warning("%s - %s", msg, self._JUSTIFICATION_LINK)
         _LOGGER.warning("Available configurations:")
 
-        configurations = self.context.graph.get_solved_python_package_versions_software_environment_all()
+        solvers = self._THOTH_ADVISER_DEPLOYMENT_CONFIGURED_SOLVERS.split(" ")
```

Comment:
We split on new line (each line represents an entry):

```suggestion
        solvers = self._THOTH_ADVISER_DEPLOYMENT_CONFIGURED_SOLVERS.split()
```

Reply:
I'll correct that, thanks!